Scripts/prepare_release.py

```diff
@@ -375,10 +375,11 @@ def prepare(self):
         progress("Done!")
         progress()
 
-        DEEPTESTS_URL = "https://github.com/dafny-lang/dafny/actions/workflows/deep-tests.yml"
+        DEEPTESTS_URL = "https://github.com/dafny-lang/dafny/actions/workflows/nightly-build.yml"
```

Comment:
Just noticing that the script is now wrong to tell you to trigger this, since `publish-release-reusable.yml` now runs the tests inline. But since this PR is just merging branches I don't want to fix it here, let's just fix that in master after merging this.